src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
 Copyright (c) Microsoft Corporation. All rights reserved.
 
-https://aka.ms/powershell
+https://aka.ms/pscore6-docs
```

Comment:
It was merged a day before in https://github.com/PowerShell/PowerShell/pull/9775

Reply:
Fixed